### ISSUE TITLE
Facetime "crashes"

### ISSUE THREAD

**Issue by larsdennert** [airmessage/airmessage-server-next#6]
Seems that Facetime eventually stops responding and the camera goes dead. I'm not sure if others experienced this but I have to restart facetime. I wonder if having the camera going 24 hours per day eventually causes an issue. Maybe minimizing Facetime at the end of a call might help this situation or even closing the app?

**Comment by tagavari** [comment]
Good point, AirMessage should definitely be disabling the FaceTime app while it's not in use. I'll look into this!

**Comment by larsdennert** [comment]
I'm thinking in the leaveCall script issue the following command right before the three returns:

tell application "Facetime"
      quit
end tell

This might also allow one Mac to have multiple Airmessage users enable Facetime as the camera won't be locked to one user. Alpha 8 is definitely better already but I'll report another issue separately.

**Comment by larsdennert** [comment]
New discovery. Facetime will not activate from a log in screen. The comands get queued up if you try multiple Facetime sessions and later run on top of each other as well. In fact, if Facetime was running and you go to a login screen the camera is also instantly disabled as well when you log back in. Facetime will also not activate if a screen saver is running. Maybe there is a way to Applescript out of the screen saver but easier to disable it entirely.

MacOS is introducing some caveats here.
